velocidi-sdk/src/main/kotlin/com/velocidi/Util.kt

```diff
@@ -75,17 +74,12 @@ object Util {
      * @return URL with the new parameters
      */
     @Throws(URISyntaxException::class)
```

Comment:
Does it still throw this type?

Reply:
Now, this function doesn't throw. Removed in e34a461